catalog.spec

```diff
@@ -284,5 +284,40 @@ module Catalog {
     funcdef approve_developer(string username) returns () authentication required;
     funcdef revoke_developer(string username) returns () authentication required;
 
+    /*
+        user_id - GlobusOnline login of invoker,
+        method_name - fully qualified method name (including module name and slash),
```

Comment:
lets name this app_id to be future proof.  Also using the word 'name' for methods/apps should be saved just for nice display names I think.